app/controllers/users_controller.rb

```diff
@@ -1,2 +1,12 @@
 class UsersController < ApplicationController
+  before_action :authorize_user
```

Comment:
It looks like this code is replicated from the admin_users_controller. Does this need to be deleted?